spec/e2e/id_property_spec.rb

```diff
@@ -64,7 +64,7 @@
           Neo4j::Config[:id_property] = :the_id
           Neo4j::Config[:id_property_type] = :auto
           Neo4j::Config[:id_property_type_value] = :uuid
-          stub_const('Clazz', Class.new do
+          stub_const('Clazz', UniqueClass.create do
```

Comment:
Yeah, agreed.  Though I imagine we could remove all indexes/constraints before each full suite run and still get benefits of not having old ones lying around